tcms/testruns/views.py

```diff
@@ -1416,21 +1412,17 @@ def change(self):
             if chk_perm:
                 return HttpResponse(json.dumps(chk_perm))
 
-            try:
-                for tr in self.trs:
-                    tr.remove_env_value(env_value=self.get_env_value(
-                        request.GET.get('old_env_value_id')
-                    ))
-
-                    tr.add_env_value(env_value=self.get_env_value(
-                        request.GET.get('new_env_value_id')
-                    ))
-            except:
-                raise
+            for test_run in self.test_runs:
+                test_run.remove_env_value(env_value=self.get_env_value(
+                    request.GET.get('old_env_value_id')
+                ))
+                test_run.add_env_value(env_value=self.get_env_value(
+                    request.GET.get('new_env_value_id')
+                ))
```

Comment:
Just FYI,
I'm wondering why the heck we try to catch exceptions above when doing `remove_env_value` and here we don't. Let's leave it as-is but definitely looks fishy.